spec/features/update_todo_items_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.feature "UpdateTodoItems", type: :feature do
+  it "complete a todo item" do
```

Comment:
One of my mentors told me early on to make my docstrings read like a sentence. E.g.

```
RSpec.feature "Update Todo Items" do
  context "User checks complete box" do
     scenario "task status updates to completed" do
       ...
       ...
       ...
       expect(page). to have_css(...)
    end
  end
end
```

The effect is that the output is exceptionally readable with the testing output.